From: Julian Ospald <hasufell@gentoo.org>
Date: Thu Aug 16 22:25:12 UTC 2012
Subject: build system

fix compilation for linkers that don't permit underlinking
reorder linker line to fix issues with as-needed

--- pokerth_server.pro
+++ pokerth_server.pro
@@ -114,8 +114,8 @@
 LIBS += -lpokerth_lib \
 	-lpokerth_db \
 	-lpokerth_protocol \
-	-lcurl \
-	-lircclient
+	-lircclient \
+	$$system(pkg-config --libs --static libcurl)
 
 win32 {
 	DEFINES += CURL_STATICLIB
